packages/payload/src/fields/setDefaultBeforeDuplicate.ts

```diff
@@ -1,13 +1,14 @@
 // @ts-strict-ignore
 // default beforeDuplicate hook for required and unique fields
 import { type FieldAffectingData, type FieldHook, fieldShouldBeLocalized } from './config/types.js'
+import { handleUniqueFields } from './utilities/handleUniqueFields.ts.js'
 
-const unique: FieldHook = ({ value }) => (typeof value === 'string' ? `${value} - Copy` : undefined)
+const hasValue = (value) => typeof value === 'string' && value.trim() !== ''
```

Comment:
`hasValue` is a little generic, can we use `isStringValue` or similar?

Reply:
Updated to `isStringValue`